inspector-axon/src/main/java/io/axoniq/inspector/utils.kt

```diff
@@ -0,0 +1,40 @@
+package io.axoniq.inspector
+
+import org.axonframework.common.ReflectionUtils
+import java.lang.reflect.Field
+import kotlin.reflect.KClass
+
+/*
+ * Copyright (c) 2022-2023. Inspector Axon
```

Comment:
Nit: doesn't the copyright notice need to come at the top of the file i.o. after the package and import statements? :sweat_smile: